WordPress/Classes/Utility/Media/MediaImageExporter.swift

```diff
@@ -117,24 +121,24 @@ class MediaImageExporter: MediaExporter {
     /// - parameter onCompletion: Called on successful export, with the local file URL of the exported UIImage.
     /// - parameter onError: Called if an error was encountered during creation.
     ///
-    func exportImageSource(_ source: CGImageSource, filename: String?, type: String, onCompletion: @escaping (MediaImageExport) -> (), onError: @escaping (MediaExportError) -> ()) {
+    func exportImageSource(_ source: CGImageSource, filename: String?, type: CFString, onCompletion: @escaping (MediaImageExport) -> (), onError: @escaping (MediaExportError) -> ()) {
         do {
             let filename = filename ?? defaultImageFilename
             // Make a new URL within the local Media directory
             let url = try MediaLibrary.makeLocalMediaURL(withFilename: filename,
-                                                         fileExtension: fileExtensionForUTType(type),
+                                                         fileExtension: String.fileExtensionForUTType(type),
```

Comment:
That's kind of the awkwardness of moving this into an extension, it really doesn't do much, and it doesn't rely on any particular class, such as FileManager.